### ISSUE TITLE
A few questions from a new bspwm user

### ISSUE THREAD

**Issue by catzybluphish** [baskerville/bspwm#873]
1. How can I change the super key to alt?
2. How can I spawn new windows on the right of my current window like in i3?
3. This may be only related to polybar but when I launch it, it'll show a thin gap between the bottom of the bar and the top of the window. However, if I start bspwm without a bar (only tried polybar) it'll take up all the screen space without the gap at the top. See the [post](https://github.com/jaagr/polybar/issues/1504) I made on their page (bspwmrc at bottom). I thought putting ` bspc config top_padding 0` would help but doesn't.
Any help would be greatly appreciated!

**Comment by catzybluphish** [comment]
The third question has been solved!

**Comment by ikselven** [comment]
About the first question:
AFAIK you'd do that in the keyboard configuration for X11. As a starting point, you might want to read [this section of the Arch Wiki](https://wiki.archlinux.org/index.php/Xorg/Keyboard_configuration#Swapping_Caps_Lock_with_Left_Control) and adapt it to your situation.

**Comment by msteen** [comment]
1. Like @ikselven mentioned, if you want to change it on a global scale, you have to look into how you can configure X11 to make it so. [Here is a StackExchange question about it.](https://unix.stackexchange.com/questions/86933/swap-alt-and-super) If instead you only want to change the bindings used by bspwm, you could just replace super for alt in your [sxhkdrc file](https://github.com/baskerville/sxhkd).
1. There are many ways to achieve something like that. If you spawn windows with dmenu or rofi, you could modify them to run: `bspc node --presel-dir east` behorehand, for example. Or maybe you wanted to spawn the window on the longest side, then you could set the automatic mode to do so, like this: `bspc config automatic_scheme longest_side`. Another solution is to try and get your behavior by [leveraging `external_rules_command`](https://github.com/baskerville/bspwm/blob/bc3649fc372e043f5e64ee76bc633d0ab13ea775/doc/bspwm.1.asciidoc#global-settings). Here is a [(probably outdated) example](https://github.com/baskerville/bspwm/blob/c9925de1825cd5701ab6b09af9626a5622bb9172/examples/external_rules/pseudo_automatic_mode/external_rules) of how you could approach it.

**Comment by catzybluphish** [comment]
@msteen @ikselven Thanks for the help but one last question. I used to be able to run urxvtd in i3 and close them normally with super + shift + q but if I do the same in bspwm, I won't be able to open urxvtc/urxvtd. I noticed that closing a program in bspwm uses `bspc node -{c,k}` which I believe means close and kill respectively. I removed the k and it urxvtd is working as it should now but I want to know if the k option is necessary.
